src/components/timelapse/DatePicker.vue

```diff
@@ -43,7 +44,8 @@ export default {
     return {
       dateFormat: "D MMM",
       dateOne: "",
-      dateTwo: ""
+      dateTwo: "",
+      endDate: format(today, "YYYY-MM-DD")
```

Comment:
To follow how `dateOne` and `dateTwo` are set, I would initialize `endDate` to `""` here and then do `this.endDate = this.dateTwo;` in the `created()` method.